### ISSUE TITLE
Fix a11y issues in Documentation/React/Dropdown

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-org#1312]
https://www.patternfly.org/v4/documentation/react/components/dropdown/

There are 2 types of a11y issues (the rest are covered by other issues)
1. Buttons must have discernable text
2. id attribute value must be unique

<img width="483" alt="Screen Shot 2019-07-17 at 10 07 57 AM" src="https://user-images.githubusercontent.com/20118816/61382428-c66cb400-a87a-11e9-9e95-0342315f3ebc.png">

Steps to reproduce the a11y errors:

1. Right click on the page and choose inspect
2. In the developer tools, choose "axe" from the top menu
3. Run the test

**Comment by christiemolloy** [comment]
@lareplog we will need to create an issue in pf-react to add an a11y label to the icon.

**Comment by jgiardino** [comment]
There are a couple of other PRs related to this work:
- Originally the ability to specify an aria-label on a dropdown toggle was included in https://github.com/patternfly/patternfly-react/pull/2517
- Then there was a regression that happened during a Typescript conversion. This PR fixes that: https://github.com/patternfly/patternfly-react/pull/2690

We realized this regression happened when reviewing https://github.com/patternfly/patternfly-react/pull/2638

However, it's worth noting here that if this issue continues to exist in the context of this repo, that you should check that aria-label is defined for the `<DropdownToggle>` component. We intentionally did not define a default for aria-label in the original PR because that string should be provided by the consumer when the consumer chooses to use an icon.
